frontend/src/components/SchemaDiagram/ER/libs/SVGLine.vue

```diff
@@ -2,59 +2,59 @@
   <svg
     version="1.1"
     xmlns="http://www.w3.org/2000/svg"
-    class="absolute"
+    class="absolute cursor-pointer"
+    pointer-events="none"
+    :viewBox="[viewBox.x, viewBox.y, viewBox.width, viewBox.height].join(' ')"
     :style="{
-      left: `${bbox.x - AA_OFFSET[0]}px`,
-      top: `${bbox.y - AA_OFFSET[1]}px`,
-      width: `${bbox.width + AA_OFFSET[0] * 2}px`,
-      height: `${bbox.height + AA_OFFSET[1] * 2}px`,
+      left: `${bbox.x + viewBox.x}px`,
+      top: `${bbox.y + viewBox.y}px`,
+      width: `${viewBox.width}px`,
+      height: `${viewBox.height}px`,
     }"
   >
     <path
+      ref="track"
       version="1.1"
       xmlns="http://www.w3.org/2000/svg"
       :d="svgLine"
-      v-bind="aaProps"
       pointer-events="visibleStroke"
       fill="none"
-      stroke="11.5"
-      stroke-width="11.5"
-      class="opacity-40"
-    ></path>
+      :stroke="hover ? 'rgba(55,48,163,0.1)' : 'transparent'"
+      :stroke-width="GLOW_WIDTH"
+    />
     <path
       v-for="(decorator, i) in svgDecorators"
       :key="i"
       version="1.1"
       xmlns="http://www.w3.org/2000/svg"
       :d="decorator"
-      v-bind="aaProps"
-      stroke="rgb(40, 40, 40)"
       fill="none"
+      :stroke="hover ? '#4f46e5' : '#1f2937'"
```

Comment:
`'#4f46e5'/'#1f2937'` is used in multiple places. It might be better to declare a computed style or constant.

Reply:
updated